content/docs/howto/swap.md

```diff
@@ -5,103 +5,103 @@ bookFlatSection: true
 
 # Swap
 
-Swap in Terraswap works as same as trade in other exchanges. Price of centralized exchanges for both stock and cryptocurrency move by price bidding system, and supply & demand. These prices change in millisecond intervals, so real-time transaction is maintained in the market. But if anyone is trying to implement an exchange with the bidding-based pricing on blockchain, immediate execution of transaction becomes impossible due to block time. 
+Swap in Terraswap works as same as trade in other exchanges. The prices of centralized exchanges for both stock and cryptocurrency move by price bidding system, and supply and demand. These prices change in millisecond intervals, and the market carries out real-time transactions. In contrast, immediate execution of transaction becomes almost impossible due to block time when it comes to the exchange on blockchain.
 
-Therefore, a different pricing approach is used in Terraswap - algorithmic pricing by tracking the ratio of the paired asset within the liquidity pool. To learn more about this approach, please refer to [mechanism section]({{< relref "/docs/introduction/mechanism" >}}).
+Therefore, Terraswap adopted a different pricing approach - algorithmic pricing by tracking the ratio of the paired asset within the liquidity pool. To learn more about this approach, please refer to [mechanism section]({{< relref "/docs/introduction/mechanism" >}}).
 
 ## Prerequisites
 
-- Token & pair should be deployed
-- You should know the addresses of token & pair.
+- Tokens and pairs should be deployed on the chain.
+- You should know the addresses of tokens and a pair for trade.
 
-## Swap by using CLI
+## Swap by Using CLI
 
-### When Offer_asset is native & IBC token
+### When offer_asset Is Native & IBC Token
 
-By command line, all transactions can be executed in this way:
+All transactions can be executed with the below command:
 
 ```bash
 terrad tx wasm execute <contract-address> <handle-msg> <coins>
 ```
 
-- `contract-address`: In swap transaction, this should be the pair address.
-- `handle-msg`: It represents the method and parameters of this execution. Will explain below.
-- `coins`: Fee to execute transaction
+- `contract-address`: The pair contract address in a swap transaction
+- `handle-msg`: The method and parameters of the execution, which will be explained following lines
+- `coins`: Transaction execution fee
 
-Enter `contract-address`, `coins` and `handle-msg`. To learn more about the general rules for `handle-msg` please refer to this [link]({{< relref "/docs/howto/query" >}}).
+To learn more about the general rules for `handle-msg`, please refer to this [link]({{< relref "/docs/howto/query" >}}).
 
 ```json
 {
-    "swap": {
-        "offer_asset": {
-            "info" : {
-                "native_token": {
-                    "denom": "uluna"
-                }
-            },
-            "amount": "10"
-        },
-        "to": "<HumanAddr>"
-    }
```

Comment:
It looks good to have tabs.

Reply:
https://github.com/DELIGHT-LABS/docs-terraswap-io/pull/38/commits/74afa8fcb06f1782d3ec545cf50cb6ca18b0c1b5